pkgs/by-name/ra/radiance/package.nix

```
@@ -0,0 +1,34 @@
+{
+  lib,
+  stdenv,
+  cmake,
+  libGL,
+  libGLU,
+  fetchFromGitHub,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "radiance";
+  version = "5.4";
+
+  src = fetchFromGitHub {
+    owner = "LBNL-ETA";
+    repo = pname;
```

Packages should avoid superfluous use of `pname` (see ##277994 for details):
```suggestion
    repo = "radiance";
```